packages/tree/src/Dendrogram.ts

```diff
@@ -154,14 +155,34 @@ export class Dendrogram extends SVGZoomWidget {
         lines.exit().remove();
 
         //  Nodes  ---
-        const textOffsetX = this.circleRadius() + 2;
+        const textOffsetX = context.labelInNode() ? 0 : this.circleRadius() + 2;
         function nodeTransform(d) {
             if (context.radial()) {
                 return "rotate(" + (d.x - 90) + ")translate(" + d.y + ")";
             }
             return context.orientation() === "horizontal" ? "translate(" + d.y + "," + d.x + ")" : "translate(" + d.x + "," + d.y + ")";
         }
+        const sizes = dataNodes
+            .map(n => {
+                return _findSize(n);
+
+                function _findSize(node) {
```

Comment:
Remove inner function - its not adding any value that I can see?

Reply:
It exists to allow recursion

Reply:
I missed that - apologies!